<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 250, 254);</string>
   </property>
   <widget class="QWidget" name="con_only_widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>251</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);
border:1px ,solid ;
border-color: rgb(0, 0, 255);
border-radius:15px;
</string>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>201</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Gestion des Membres</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>201</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string> Gestion des Livres</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>201</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Gestion des Emprunts</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>19</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Dashboard</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>601</width>
      <height>601</height>
     </rect>
    </property>
    <widget class="QWidget" name="header" native="true">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>551</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border:0.5px solid; 
border-radius:15px;
border-color: rgb(255, 255, 255);
</string>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>381</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 18pt &quot;Tempus Sans ITC&quot;;</string>
      </property>
      <property name="text">
       <string>Application pour bibliothéque</string>
      </property>
     </widget>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>571</width>
       <height>461</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QWidget" name="widget_2" native="true">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>561</width>
         <height>211</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border:1px solid;
border-radius:15px;
border-color:none;
box-shadow: 10px 5px 5px;</string>
       </property>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>191</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border:none;
color:black;</string>
        </property>
        <property name="text">
         <string>Statique des livres et membres</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>80</y>
          <width>104</width>
          <height>70</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="widget_4" native="true">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>59</y>
          <width>191</width>
          <height>101</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 255);</string>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>20</y>
           <width>111</width>
           <height>20</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
border:none;</string>
         </property>
         <property name="text">
          <string>Tous les  Livres</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>60</y>
           <width>111</width>
           <height>20</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
border:none;</string>
         </property>
         <property name="text">
          <string>            3</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="widget_5" native="true">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>60</y>
          <width>191</width>
          <height>101</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 255);</string>
        </property>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>20</y>
           <width>111</width>
           <height>20</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
border:none;</string>
         </property>
         <property name="text">
          <string>Tous les membres</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>60</y>
           <width>111</width>
           <height>20</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
border:none;</string>
         </property>
         <property name="text">
          <string>            3</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="widget_6" native="true">
       <property name="geometry">
        <rect>
         <x>19</x>
         <y>230</y>
         <width>561</width>
         <height>251</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border:1px solid;
border-radius:15px;
border-color:none;
box-shadow: 10px 5px 5px;</string>
       </property>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>201</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border:none;
color:black;</string>
        </property>
        <property name="text">
         <string>Liste des livres les plus empruntés</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="tableWidget">
        <property name="geometry">
         <rect>
          <x>15</x>
          <y>70</y>
          <width>521</width>
          <height>121</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
